examples/resources/fabric_deployment_pipeline/resource.tf

```
@@ -0,0 +1,17 @@
+resource "fabric_deployment_pipeline" "example" {
+  display_name = "Deployment Pipeline Example"
+  description  = "Deployment Pipeline Example"
+  id           = data.fabric_deployment_pipeline.example.id
+  stages = [
+    {
+      display_name = "Stage 1",
+      description  = "Stage 1",
+      is_public    = true,
```

missing option to set `workspace_id`
```suggestion
      is_public    = true,
      workspace_id = "00000000-0000-0000-0000-000000000000"
```

---

@DariuszPorowski please correct me if I am wrong, but according to the documentation, workspace id cannot be assigned during "Create" operation, its a part of another API endpoint. 

https://learn.microsoft.com/en-us/rest/api/fabric/core/deployment-pipelines/create-deployment-pipeline?tabs=HTTP 

Shoudn't stage assignment be part of another example for that endpoint?

---

@CatalinDabuleanu it does not have to be during API create operation. During the TF create operation if `workspace_id` is provided then you can do https://learn.microsoft.com/en-us/rest/api/fabric/core/deployment-pipelines/assign-workspace-to-stage